scripts/build-backend.sh

```diff
@@ -46,28 +46,38 @@ echo -en "Minifying with noderify..."
   --filter=async_hooks \
   --filter=utf-8-validate \
   --filter=bufferutil \
-  --filter=worker_threads \
   --filter=http2 \
   --filter=pino-pretty \
   --filter=memcpy \
   --filter=diagnostics_channel \
+  --filter=perf_hooks \
   index.js | \
   # Apologies to future contributors: this worker needs to load better-sqlite3,
   # and I could not get bindings() to correctly load the native code when this is
   # not at the project root. If the path name of the worker changes in the
   # original module then this replacement needs to be updated. Apologies for the
   # fragility, but this was the best solution I could find in limited time.
-  sed 's|\./lib/mbtiles_import_worker\.js|../../../../mbtiles_import_worker.js|' > \
+  sed 's|\.\./lib/mbtiles_import_worker\.js|../../../../../mbtiles_import_worker.js|' | \
+  # Additional apologies for this, which is even more fragile. Context is that 
+  # Piscina introduces its own worker file (unfortunately with a very generic name of `worker.js`)
+  # so we need to update the path it references to point to the eventually relocated
+  # file within the nodejs-project directory.
+  sed "s|'worker\.js'|'../../../../piscina_worker.js'|" > \
```

Comment:
somehow managed an even more fragile addition to this file compared to the import worker above 😄

Reply:
yeah... it will do for now. It's not crazy to do this kind of thing with regex (see https://github.com/danielgindi/rollup-plugin-natives/blob/master/src/index.js) but it is fragile, and it's more robust to do this with an AST parser e.g. acorn - see https://github.com/playcode/rollup-plugin-node-globals/blob/master/src/inject/index.js

At some point we can write some rollup plugins to do an AST parse and extract any workers, but this will do for now.